projects/Internet Connection Check/internet_connection_check.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+def internet_conntection_test():
```

Comment:
small typo mistake here 😃 , the function you are calling is different from the function you have defined

Reply:
Ah! Thanks for the heads up!